src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -30,48 +30,38 @@ import ReportActionComposeFocusManager from '@libs/ReportActionComposeFocusManag
 import * as ReportActionsUtils from '@libs/ReportActionsUtils';
 import * as ReportUtils from '@libs/ReportUtils';
 import setShouldShowComposeInputKeyboardAware from '@libs/setShouldShowComposeInputKeyboardAware';
-import reportPropTypes from '@pages/reportPropTypes';
 import * as EmojiPickerAction from '@userActions/EmojiPickerAction';
 import * as InputFocus from '@userActions/InputFocus';
 import * as Report from '@userActions/Report';
 import * as User from '@userActions/User';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {Modal, ReportAction as ReportActionType, Report as ReportType} from '@src/types/onyx';
```

Comment:
You can use `import type * as OnyxTypes from '@src/types/onyx';` instead, to avoid "Type" suffixes